docs/en/getting_started/example_datasets/ontime.md

```diff
@@ -219,7 +219,7 @@ ORDER BY c3 DESC;
 Better version of the same query:
 
 ``` sql
-SELECT Carrier, avg(DepDelay > 10) * 100 AS c3 FROM ontime WHERE Year = 2007 GROUP BY Carrier ORDER BY Carrier
+SELECT Carrier, avg(DepDelay > 10)*100 AS c3 FROM ontime WHERE Year = 2007 GROUP BY Carrier ORDER BY Carrier
```

Comment:
if we prefer that there're no spaces around binary operators here then we should remove them around `>` and `=` (though usually I personally to have them), in many places below as well

P.S. also I think these queries would look better if we add some newlines before major keywords